web/server.js

```diff
@@ -116,6 +117,14 @@ module.exports = function(options) {
       method: 'GET',
       path: '/login/oauth/access_token',
       config: {
+        validate: {
+          query: {
+            code: Joi.string(),
+            client_id: Joi.string(),
+            client_secret: Joi.string(),
+            grant_type: Joi.string()
```

Comment:
I think this should be https://github.com/hapijs/joi#anyvalidvalue---aliases-only-equal right?